app/cafe.py

```diff
@@ -0,0 +1,27 @@
+import datetime
+
+from app.errors import (
+    NotVaccinatedError,
+    OutdatedVaccineError,
+    NotWearingMaskError
+)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str | None:
+        if (
+                visitor.get("vaccine") is None
```

Comment:
Here `if not visitor.get("vaccine")` would be enough to check if visitor is vaccinated